examples/sort-axis.rs

```diff
@@ -84,6 +87,15 @@ where
         });
         perm
     }
+
+    fn sort_axis_<F>(&self, axis: Axis, mut less_than: F) -> Permutation
+    where
+        F: FnMut(usize, usize) -> Ordering,
+    {
+        let mut perm = self.identity(axis);
+        perm.indices.sort_by(move |&a, &b| less_than(a, b));
+        perm
+    }
 }
 
 impl<A, D> PermuteArray for Array<A, D>
```

Comment:
Ah, makes sense!  Now that this is resolved I think we can drop the "simple" method and the associated test.  Also may not need the big array file test either, or at least can trim the file up some to avoid bloat, it was primarily needed just to chase this down as a one off.